sdk/formrecognizer/ai-form-recognizer/src/util.ts

```diff
@@ -64,7 +67,10 @@ export function makeServiceClient(
   const authPolicy = isTokenCredential(credential)
     ? bearerTokenAuthenticationPolicy({
         credential,
-        scopes: DEFAULT_COGNITIVE_SCOPE,
+        scopes: [options.audience ?? DEFAULT_COGNITIVE_SCOPE].map((scope) => {
```

Comment:
Is this bug from? end users can't specify the scopes?

Reply:
they'll specify the audience, this is an old data-plane package, and this is the last version of this package. Not a bug.